zephyr/Makefile

```diff
@@ -71,15 +70,10 @@ $(GENERIC_TARGETS): $(LIBMICROPYTHON)
 $(CLEAN_TARGETS):  clean
 
 $(GENERIC_TARGETS) $(KCONFIG_TARGETS) $(CLEAN_TARGETS):
-	$(RM) -f outdir/$(OUTDIR_PREFIX)/zephyr.lnk
```

Comment:
What has replaced this?

We've discussed before that doing the $(RM) here is over-zealous but we do need to remove zephyr.lnk when libmicropython.a is rebuilt and I can't see any logic to do that. Without that then when we change uPy C files they won't make it into the final zephyr application.